src/sagemaker/tensorflow/README.rst

```diff
@@ -207,6 +207,63 @@ After attaching, the estimator can be deployed as usual.
 
     tf_estimator = TensorFlow.attach(training_job_name=training_job_name)
 
+Distributed Training
+''''''''''''''''''''
+
+To run your training job with multiple instances in a distributed fashion you need to set ``train_instance_count``
+to a number larger than 1. We support two different types of distributed training, parameter server and Horovod.
```

Comment:
is running a script that uses MPI but not Horovod a use case?

Reply:
I think it is but if you use the tensorflow container it uses Horovod. I could be wrong.

Reply:
Yes, MPI without Horovod is a valid use case.

Reply:
if that's the case, then I think this should be changed to something like "We support two different ways of handling distributed training: parameter servers and MPI. The use of MPI can be with or without Horovod." maybe include a link to Horovod documentation as well.